Examples/Examples/WorldScaleGeoTrackingExampleView.swift

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+import CoreLocation
+
+/// An example that utilizes the `WorldScaleGeoTrackingSceneView` to show an augmented reality view
+/// of your current location. Because this is an example that can be run from anywhere,
+/// it places a red circle around your initial location which can be explored.
+struct WorldScaleGeoTrackingExampleView: View {
+    @State private var scene: ArcGIS.Scene = {
+        // Creates an elevation source from Terrain3D REST service.
+        let elevationServiceURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
+        let elevationSource = ArcGISTiledElevationSource(url: elevationServiceURL)
+        let surface = Surface()
+        surface.addElevationSource(elevationSource)
+        let scene = Scene()
+        scene.baseSurface = surface
+        scene.baseSurface.backgroundGrid.isVisible = false
+        scene.baseSurface.navigationConstraint = .unconstrained
+        scene.basemap = Basemap(style: .arcGISImagery)
+        scene.addOperationalLayer(parcelsLayer)
+        return scene
+    }()
+    
+    /// Basemap opacity.
+    @State private var opacity: Float = 1
+    /// Graphics overlay to show a graphic around your initial location.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    /// The location datasource that is used to access the device location.
+    @State private var locationDatasSource = SystemLocationDataSource()
```

Comment:
Should we pass this LDS to the `WorldScaleGeoTrackingSceneView` so that it does not create a second LDS in its default init parameters?